src/components/Profile/Profile.js

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+// import { Spinner } from 'react-spinners-css';
+import style from './Profile.module.css';
+import { displayMissionFunction } from '../../redux/mission/Mission';
+import { getRocketFromAPI } from '../../redux/Rockets/rockets';
+
+const propTypes = {};
+
+const defaultProps = {};
+
+const Profile = () => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(displayMissionFunction());
+    dispatch(getRocketFromAPI());
+  }, []);
+
+  const missionList = useSelector((state) => state.missionReducer);
+  const rocketsList = useSelector((state) => state.RocketsReducer);
+
+  return (
+
+    <div className={style.myprofile}>
+      <div className={style.mymission}>
+        <h1>My Mission</h1>
+        <ul className={style.table}>
+          {missionList.map((el) => (
+            <li key={el.id}><p>{el.name}</p></li>
+          ))}
+        </ul>
+      </div>
```

Comment:
- [OPTIONAL] The `My Profile` section can be enhanced by adding a placeholder message when the "My Missions" or "My Rockets" lists are empty (no missions joined or no rockets reserved).
- [OPTIONAL] The `My Profile` section can be enhanced by adding the "Cancel reservation" and "Leave Mission" buttons to the lists here. Clicking them should dispatch the actions you have already used in the main Rockets and Missions sections.
- [OPTIONAL]  The `My Profile` section can be enhanced by adding the "Read more" button for each mission and rocket. Upon click, it should open a corresponding Wikipedia page in a new tab. NOTE - you need to get that extra Wikipedia URL from the API's payload.

- Kindly add this to improve you application 👍🏻 💯